src/transports/dash/manifest_parser.ts

```diff
@@ -71,16 +71,16 @@ export default function generateManifestParser(
     const { response, scheduleRequest } = args;
     const argClockOffset = args.externalClockOffset;
     const loaderURL = args.url;
-    const url = response.url == null ? loaderURL :
-                                       response.url;
+    const url = response.url === undefined ? loaderURL :
```

Comment:
Why not :
`const url  = response.url ?? loaderURL;`

Reply:
Done